guidelines/content-guidelines/09-add-documentation-topic.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Add new documentation to the website
+---
+​
+This document explains how to add a new topic to the Kyma documentation to render it on the [Kyma website](https://kyma-project.io). It also describes how to modify the existing topic if you want OpenAPI specifications to show in the given topic documentation.
+​
+## Add a new documentation topic
+​
+Follow these steps:
+​
+1. Create a pull request that contains the `.md` files for the new documentation topic under the new `docs` subfolder in the `kyma` repository, such as `docs/api-gateway-v2/`.
+​
+2. In the same PR, create a `yaml` file under the [`templates`](https://github.com/kyma-project/kyma/tree/master/resources/core/charts/docs/charts/content-ui/templates) folder to add a [ClusterDocsTopic CR](https://kyma-project.io/docs/components/headless-cms/#custom-resource-cluster-docs-topic) CR for your topic. For example, if you add a ClusterDocsTopic CR for the API Gateway v2 component, name it `docs-components-api-gateway-v2-cdt.yaml`.
+​
+See the example definition:
+​
+``` yaml
+apiVersion: cms.kyma-project.io/v1alpha1
+kind: ClusterDocsTopic
+metadata:
+  labels:
+    cms.kyma-project.io/view-context: docs-ui
+    cms.kyma-project.io/group-name: components
+    cms.kyma-project.io/order: "11"
+  name: api-gateway-v2
+spec:
+  displayName: "API Gateway v2"
+  description: "Overall documentation for API Gateway v2"
+  sources:
+    - type: markdown
+      name: docs
+      mode: package
+      url: https://github.com/{{ .Values.global.kymaOrgName }}/kyma/archive/{{ .Values.global.docs.clusterDocsTopicsVersion }}.zip
+      filter: /docs/api-gateway-v2/
+```
+​
+3. Adjust values for these fields:
+​
+- **cms.kyma-project.io/order** defines the order for your topic in the left navigation, such as `"11"`. To add it correctly, check in other `yaml` files which number is assigned to the last topic in the documentation navigation on the website, and add a consecutive number to your component. If you decide to modify the existing topic order, change the values for this parameter in all other `yaml` files accordingly to avoid duplicates.
```

Comment:
```suggestion
- **cms.kyma-project.io/order** defines the number of your topic on the list in the left navigation, such as `"11"`. To add it correctly, check in other `.yaml` files which number is assigned to the last documentation topic in the navigation on the website, and add a consecutive number to your component. If you decide to modify the existing topic order, change values for this parameter in all other `.yaml` files accordingly to avoid duplicates.
```